CHANGELOG.md

```diff
@@ -3188,6 +3234,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/).
 
 Hello, OpenDAL!
 
+[v0.44.1]: https://github.com/apache/incubator-opendal/compare/v0.44.0...v0.44.1
+[v0.44.0]: https://github.com/apache/incubator-opendal/compare/v0.43.0...v0.44.0
```

Comment:
Out of curiosity, why "0.44.0" added here instead of when we were releasing 0.44.0?

Reply:
> Out of curiosity, why "0.44.0" added here instead of when we were releasing 0.44.0?

It was missed during the last update.